src/app/not-found.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from '@/components/Button'
+
+import { PATHS } from './_constants/paths'
+
+export default function NotFound() {
+  return (
+    <div className="m-auto flex max-w-xs flex-col gap-6 py-[104px] sm:items-center sm:text-center">
```

Comment:
Are you following up with Filipa regarding [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/84#discussion_r1580669681)? 🙏🏼